webapp/views/wpt-results.js

```diff
@@ -871,6 +876,11 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
   }
 
   canAmendMetadata(node, index, testRun) {
+    // It is always possible in triage mode to amend metadata for a problem
+    // with a test file itself.
+    if (index == null)
```

Comment:
Lets add '{}' around this `if (index == null)`

Reply:
Done.